README.md

```diff
@@ -1,6 +1,6 @@
 # FFC Template Node
 
-Template to support rapid delivery of microservices for FFC Platform. It contains the configuration needed to deploy a simple Hapi Node server to the Azure Kubernetes Platform.
+Template to support rapid delivery of microservices for FFC Platform. It contains the configuration needed to deploy a simple Hapi Node server to the Azure Kubernetes Platform. 
```

Comment:
intended?

Reply:
Yes, but only because i needed to trigger another build and don't have permission to run via Jenkins. I needed to trigger due to the following.. _Build aborted - not a PR or a main branch_

Not had this issue before but will dig deeper next feature branch I create. I will push a change to remove the space now.

Reply:
👍